GameData/B9_Aerospace_HX/Patches/Classic/B9_Aerospace_HX-Reactor-Compat.txt

```diff
@@ -0,0 +1,153 @@
+@PART[B9_Structure_HX1_G]:NEEDS[!DSEV]
```

Comment:
I think this needs some more needs

Reply:
Nah. It doesn't need more needs. When the WBI play mode is switched to Classic Stock and this becomes an active .cfg, this will be the default behavior of the reactor. Near Future Electrical, if present, should not interfere with it, and I can't see KSPI interfering with it (yet).

Reply:
Ah, that weird stuff with WBI renaming files?

Reply:
Yes, that. :smile: